amico/models.py

```diff
@@ -231,27 +226,30 @@ def set_solver( self ) :
 
     def generate( self, out_path, aux, idx_in, idx_out, ndirs ) :
         scheme_high = amico.lut.create_high_resolution_scheme( self.scheme, b_scale=1 )
-        gtab = gradient_table( scheme_high.b, scheme_high.raw[:,0:3] )
+
+        stick = Stick(scheme=scheme_high)
+        zeppelin = Zeppelin(scheme=scheme_high)
+        ball = Ball(scheme=scheme_high)
 
         nATOMS = 1 + len(self.d_perps_zep) + len(self.d_isos)
         idx = 0
         with tqdm(total=nATOMS, ncols=70, bar_format='   |{bar}| {percentage:4.1f}%', disable=(get_verbose()<3)) as progress:
             # Stick
-            signal = single_tensor( gtab, evals=[self.d_perp, self.d_perp, self.d_par] )
+            signal = stick.get_signal(diff=self.d_par, theta=0, phi=0)
```

Comment:
For all models, can we define the two parameters `theta` and `phi` to default to 0, so we don't need to specify them in the calls? At least, this is true in this specific example, but of course someone else might want to use these functions to generate signals in different orientations